testStrategies/commonToAllTestStrategies.go

```diff
@@ -218,7 +215,6 @@ func (testDefinition *TestDefinition) BuildAndSendRequest(
 	targetHost string,
 	targetPort string,
 	uniqueTestRunID string,
-	globalsMap GlobalsMaps,
 ) int64 {
```

Comment:
GlobalsMap is now actually global. No need to pass it through the stack (see changes below as well).